libs/librepcb/common/systeminfo.cpp

```diff
@@ -233,6 +236,26 @@ QString SystemInfo::getProcessNameByPid(qint64 pid) {
   }
   buf[len] = 0;
   processName = QFileInfo(QFile::decodeName(buf)).fileName();
+#elif defined(Q_OS_OPENBSD)
+  // https://man.openbsd.org/sysctl.2
+  // NOTE: This will return only the first 16 bytes of the process name. If
+  // someone finds a way to get the full process name, feel free to improve it.
+  kinfo_proc proc;
+  size_t procSize = sizeof(proc);
+  int mib[6] = {CTL_KERN,     KERN_PROC, KERN_PROC_PID, static_cast<int>(pid),
+                sizeof(proc), 1};
```

Comment:
Just wondering, is this weird indentation on purpose? :slightly_smiling_face:

Reply:
It's the output of clang-format, probably there is still some kind of alignment enabled in its rules...